lib/puppet_references/util.rb

```diff
@@ -120,5 +120,14 @@ def self.link_release_notes_if_applicable(component, text, version = nil, agent_
       end
     end
 
+    def self.convert_man(man_filepath)
+      require 'pandoc-ruby'
+      text = IO.read(man_filepath)
+      PandocRuby.convert(text, from: :man, to: :markdown)
```

Comment:
I think this can be written in a single line as `PandocRuby.new([man_filepath], from: :man).to_markdown`